astronomer/providers/apache/hive/sensors/hive_partition.py

```diff
@@ -69,5 +70,4 @@ def execute_complete(self, context: Context, event: Optional[Dict[str, str]] = N
                     "Success criteria met. Found partition %s in table: %s", self.partition, self.table
                 )
                 return event["message"]
-            raise AirflowException(event["message"])
-        raise AirflowException("No event received in trigger callback")
+            handle_error(self.soft_fail, event["message"])
```

Comment:
I thought of removing this because 1) it is not consistent in the repo i.e some places we raise errors and someplace we mark success 2) an empty event from trigger should not be possible until we yield empty